books.rb

```diff
@@ -3,8 +3,8 @@
 class Books < Item
   attr_accessor :publisher, :cover_state
 
-  def initialize(publisher, cover_state, publish_date, archived: flase)
-    super(publish_date, archived: archived)
+  def initialize(publisher, cover_state)
```

Comment:
Optional: Don't we also need to add publish_date as an argument for initialize?

Reply:
yeah  i have  done that